### ISSUE TITLE
Not able to access the "Delete" button using the mouse or the keyboard: A11y_.NET 9 template_.NET 9 template_Task_Functionality.

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26215]
### GitHub Tags
#A11yTCS; #Win32; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #Functionalty; #A11yFunctional: 

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till any existing task, and press enter on it and task screen will open.
4. Observe that not able to access the "Delete" button using the mouse or the keyboard.

https://github.com/user-attachments/assets/2b97510a-ce18-4f63-95f2-5b3930680bb2

### Actual Result:
Not able to access the "Delete" button using the mouse or the keyboard.

### Expected Result:
User should be able to access the "Delete" button using the mouse and the keyboard.

### User Impact:
Users won't be able to access the "Delete" button functionality.

### Attachment:
https://github.com/user-attachments/assets/7aad7464-32ea-4636-88f8-a796798e5474

**Comment by AnKushSingh05** [comment]
Issue is fixed, hence closing the bug

https://github.com/user-attachments/assets/38cd8ffa-61b0-4a58-ae19-0160a5524b93
